src/gui/transferlistmodel.cpp

```diff
@@ -709,46 +709,40 @@ QColor getDefaultColorByState(const BitTorrent::TorrentState state)
     case BitTorrent::TorrentState::DownloadingMetadata:
     case BitTorrent::TorrentState::ForcedDownloadingMetadata:
         if (!dark)
-            return {34, 139, 34}; // Forest Green
+            return {15, 187, 88}; // Deep Green
         else
-            return {50, 205, 50}; // Lime Green
+            return {108, 194, 74}; // Light Green
     case BitTorrent::TorrentState::StalledDownloading:
     case BitTorrent::TorrentState::StalledUploading:
         if (!dark)
             return {0, 0, 0}; // Black
         else
-            return {204, 204, 204}; // Gray 80
+            return {44, 44, 44}; // Gray
```

Comment:
It's too dark:

![color](https://user-images.githubusercontent.com/6451685/133152790-d09f382c-fd0d-4629-a725-c2094fb24283.JPG)

Reply:
On the stalled color text icon issues:

RGB 44, 44, 44 (#2C2C2C) is not `GRAY` as comments suggest it is known as `NIGHT RIDER`
Ref.: https://www.htmlcsscolor.com/hex/2C2C2C

Suggestion:
`DimGray`
RGB 105, 105, 105
Hex (#696969)
Ref.: https://www.htmlcsscolor.com/hex/696969

I would probably also change line below to reflect where you got your colors from:
https://github.com/qbittorrent/qBittorrent/blob/407471c5ba49d63a3ce2e71d2498ec793ff8451f/src/gui/transferlistmodel.cpp#L702

I personally use: https://www.htmlcsscolor.com/html-color-names or when picking a variant/web safe color  https://www.htmlcsscolor.com/

Please amend all color comments accordingly where necessary.

Reply:
I have mainly taken inspiration from the icon of Google and Microsoft. I have used [this](https://material.io/resources/color/) resource to pick neutral color.